Destination/StreamDestination.php

```diff
@@ -9,8 +10,16 @@
  */
 class StreamDestination implements Destination
 {
+    /**
+     * @var string Path to directory.
+     */
     private $directory;
 
+    /**
+     * @var array<Backup> List of backups.
```

Comment:
Can you change to `Backup[]` instead? Admittedly, I like your way better but the rest of the bundle uses `Class[]`.